RELEASING.md

```diff
@@ -0,0 +1,53 @@
+This is the Release plan and TODO list for CDT.
+
+## Steps for Release
+
+Items at the beginning of development
+
+
+Prepare main branch for next release
```

Comment:
Should we add creation of the new N&N page to this list? It seems useful to create it early so that we can add to it throughout the release cycle.